### ISSUE TITLE
Terrain scene will not load 

### ISSUE THREAD

**Issue by rslonake** [Zylann/godot_heightmap_plugin#438]
Started this morning... I have been working on this terrain for months and now when I try to open the terrain.tscn I get the following error...

"No loader found for resource: res://terrain_data/data.hterrain (expected type: )"

I pasted the text of my data.hterrain file below... it looks empty. I opened an older version of the project and the data.hterrain is much larger... it looks like the attached file is corrupted???

Is it safe to say I have lost all my work since my last version save?

Here is the contents of my data.hterrain file:

{
	"maps": [
		[
			{
				"id": 0
			}
		],
		[
			{
				"id": 0
			}
		],
		[
			{
				"id": 0
			}
		],
		[
			{
				"id": 0
			}
		],
		[],
		[],
		[],
		[]
	],
	"version": "0.11"
}

**Comment by rslonake** [comment]
Thanks for the feedback. I will keep investigating. By the way, I really like this plug-in and thank you for all the time that you've spent working on it. And for getting back to me so quickly about this issue.

**Comment by rslonake** [comment]
Quick question... as I mentioned, I have an older version of this project on my computer. I believe most of the changes are not terrain related. If I drag my terrain data folder from the old version to the new version do you think that it should load?  

In other words, all terrain data is in the terrain data folder, correct?

**Comment by Zylann** [comment]
> all terrain data is in the terrain data folder, correct?

Yes. Properties such as selected shader or LOD distance are stored in the scene, but the actual terrain data such as heights, splatmaps or detail layer maps, are stored in the terrain data folder.

**Comment by rslonake** [comment]
Good news... I was able to get my terrain back. I was just about to give up and scrap the project and go back to my previous version, but I thought, "Let's try one last thing." I deleted the terrain addon and reinstalled... readded the hterrain node and there it was! Just needed to rescale and add reimport the textures. (The texture maps were there but everything was white - once I reimported the textures it filled in the map).

Thanks again for your advice and help with this.
